conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -44,23 +38,17 @@ install:
         Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
           throw "There are newer queued builds for this pull request, failing early." }
 
-    # Cywing's git breaks conda-build. (See https://github.com/conda-forge/conda-smithy-feedstock/pull/2.)
-    - cmd: rmdir C:\cygwin /s /q
```

Comment:
Well the CI knowledge needs to live somewhere. Is it bad for that somewhere to be `conda-forge-build-setup`? If not there, where should it live?